cms/static/cms/js/modules/cms.page_type_select.js

```diff
@@ -0,0 +1,26 @@
+//##############################################################################
+// PAGE WIZARD
+
+/* global CMS */
+(function ($) {
+    'use strict';
+    // CMS.$ will be passed for $
+    $(document).ready(function () {
```

Comment:
we use the shorthand now:

```
// shorthand for jQuery(document).ready();
$(function () {
```

at best copy style/structure from: https://github.com/divio/django-cms/blob/develop/cms/static/cms/js/modules/cms.messages.js#L1